dynamic_programming/longest_common_string.cpp

```diff
@@ -1,53 +1,142 @@
-#include <iosrteam>
-using namespace std;
+/**
+ * @file
+ * @brief contains the definition of the function longest_common_string_length
+ * @details
+ * the function longest_common_string_length computes the length
+ * of the longest common string which can be created of two input strings
+ * by removing characters from them
```

Comment:
You helped in cleaning up the whole file, so you should be counted as the author as well. 🙂
```suggestion
 * by removing characters from them
 *
 * @author [Nikhil Arora](https://github.com/nikhilarora068)
 * @author [Piotr Idzik](https://github.com/vil02)
```